### ISSUE TITLE
Tooling

### ISSUE THREAD

**Issue by tripledoublev** [hyphacoop/governance-experiment#7]
# Tools Used for the Experiment

- Matrix chat
- Earthstar voting tool
- Solidity smart contract
- Python scripts

These tools have been chosen for their simplicity. Some of these will be used only by myself, such as the Python script for generating visuals and parsing meeting notes. Other tools will be used by all participants to discuss, argue, propose and vote on what makes the cut as official Hypha lore.

# Matrix

Matrix will be central to these experiments. I will create a new room to host this project's discussions and deliberations. We will use chat to describe the project, gather suggestions, and deliberate on what elements need to be canonized. For quick votes, we will use emoji reactions in Matrix. For votes that are more controversial, we will use the Earthstar voting tool I built. Once items and their metadata (image and description) are collectively approved, they will be minted on-chain as NFTs.

# On-chain
For this initial experiment, I suggest 2 options. Either we mint the tokens on testnet or on Polygon. Testnet will be free and might reduce some financial pressure. Alternatively, if we choose to go with Polygon, we can consider the option of minting these tokens as collectibles for the broader public. Orb seems like an interesting option where these items could exist for the public to collect (Orb being a web3 social platform part of the Lens protocol).

In the end, these options are not mutually exclusive. We could run the governance experiment on testnet and also mint them as collectible NFTs on another chain.

In summary, these NFTs could exist both on Ethereum testnet or on a cheap EVM chain such as Polygon. I decided to keep votes and other decisions off-chain for this early phase. Instead of having the entire process live on the blockchain, I opted to only bring its final output on-chain. This means that the prior steps are designed as a mechanism for deciding what we want to bring on-chain.

I imagine this will simplify the process for everyone and will allow us to focus on the decision-making process itself instead of technical details such as tokens, chains, and wallet interactions.

**Comment by tripledoublev** [comment]
Archived [here](https://github.com/hyphacoop/governance-experiment/blob/main/experiments/tooling.md)
